src/processor.ts

```diff
@@ -56,8 +56,9 @@ export default function createProcessor(
     resultSet: RowObject[],
     keysToResolve: string[]
   ) {
-    const resultsRequests = resultSet.map(async (row: Record<string, any>) => {
-      const resolvedRow: RowObject = {};
+    const resultsRequests = resultSet.map(async (row: RowObject) => {
+      // needs to be `any` since the resolver is being plugged into this package
+      const resolvedRow: any = {};
```

Comment:
The tests were not working with the typescript compiler because this type was `[key: string]: string | number;` and different processors return different types than `string | number`